test/parallel/test-net-better-error-messages-port-hostname.js

```diff
@@ -3,12 +3,18 @@ const common = require('../common');
 const net = require('net');
 const assert = require('assert');
 
-const c = net.createConnection(common.PORT, '***');
+const s = net.createServer();
+s.listen(0);
+const port = s.address().port;
```

Comment:
Since the `createConnection()` never gets a proper hostname, the actual connection never happens, so we don't need to carefully choose an available port, I don't think. I think you can just hardcode any valid port (e.g., `80`) and that should be fine. Maybe include a comment explaining that the port needs to be valid but is never in fact used because a connection never gets created because the hostname is invalid.